components/skus/renderer/skus_page_controller.h

```diff
@@ -0,0 +1,98 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_SKUS_RENDERER_SKUS_PAGE_CONTROLLER_H_
+#define BRAVE_COMPONENTS_SKUS_RENDERER_SKUS_PAGE_CONTROLLER_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "brave/components/skus/common/skus_sdk.mojom.h"
+#include "content/public/renderer/render_frame.h"
+#include "content/public/renderer/render_frame_observer.h"
+#include "mojo/public/cpp/bindings/remote.h"
+#include "url/gurl.h"
+#include "v8/include/v8.h"
+
+namespace skus {
+
+// If present, this will inject a few methods (used by SKU SDK)
+// into window.chrome.braveSkus.*
+//
+// This is only intended to be used on account.brave.com and the dev / staging
+// counterparts. The accounts website will use this if present which allows a
+// safe way for the browser to intercept credentials which are used in the
+// browser.
+//
+// The first use-case for this credential redemption is with VPN. Folks
+// will be able to purchase VPN from account.brave.com and the browser can
+// detect the purchase and use those credentials during authentication when
+// establishing a connection to our partner providing the VPN service.
+//
+// TODO(bsclifton): trying to cleanup the gin.
+// Looking at chrome/renderer/net/net_error_page_controller.h as an example
+class SkusPageController {
```

Comment:
This naming is inconsistent with the others and should be SkusJSHandler. Also is there a follow-up issue for converting to gin?

Reply:
Fixed in https://github.com/brave/brave-core/pull/10847/commits/f1fd81200a1006b5c0971b6805968acf4934c79a